parser.c

```diff
@@ -275,6 +274,13 @@ void process_pihole_log(int file)
 				break;
 		}
 
+		// Get timestamp
+		int querytimestamp, overTimetimestamp;
+		extracttimestamp(readbuffer, &querytimestamp, &overTimetimestamp);
+
+		// Check if this query has already been imported from the database
+		if(querytimestamp < lastDBimportedtimestamp) continue;
```

Comment:
There are more of this line below, but since this is run on every line, the others would just be redundant?

Reply:
Oh yes, they shouldn't be there, seems like I forgot to remove them at some places.